src/locales/it.json

```
@@ -73,7 +73,12 @@
         },
         "title": "Segnala un errore",
         "cancel": "Annulla",
-        "send": "Invia"
+        "send": "Invia",
+        "confirmation": {
+            "title": "Confirma l'invio",
+            "warningText": "Error reporting is not intended to order a review or consultation. To order appointment, contact your chosen doctor.",
+            "text": "Are you sure you want to report an error?"
+        }
```

The translations should not be present at all:
```suggestion
        "send": "Invia"
```
so that they are marked as "untranslated" in weblate, but until then the default (English) texts will be shown.

Also you can use this suggestion by clicking "Commit suggestion" button bellow ⬇️ this comment 😄